CHANGELOG.md

```diff
@@ -4,7 +4,22 @@ Please consider [donating](https://docs.fleaflet.dev/supporters#support-us) or [
 
 This CHANGELOG does not include every commit and/or PR - it is a hand picked selection of the ones that have an effect on you. For a full list of changes, please check the GitHub repository releases/tags.
 
-## [8.0.0] - 2025/XX/XX
+## [8.0.1] - 2025/02/XX
+
+Contains the following user-affecting changes:
+
+- Added multi-world support for `Polygon`s and `Polyline`s - [#2033](https://github.com/fleaflet/flutter_map/pull/2033)
```

Comment:
Shouldn't the version be 8.1.0 if the release contains a new feature?

Reply:
I would say not necessarily, since it is just a continued implementation of a headline feature in the last version. It's a new part of a feature. But I have no real strong feelings.

Reply:
Is this thread is blocked by my response? 👀 
A "continued implementation of a headline feature in the last version" sadly does not exist for semantic versioning, so it would be either a bug or feature. Unless we want to ignore the versioning scheme?

Reply:
> Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backward compatible functionality is introduced to the public API. It MUST be incremented if any public API functionality is marked as deprecated. It MAY be incremented if substantial new functionality or improvements are introduced within the private code. It MAY include patch level changes. Patch version MUST be reset to 0 when minor version is incremented.

Minor versions must be incremented only if the public API is changed or the public API functionality is marked as deprecated.

I guess it is true that parts of the public API have been changed, so I'll change it.